cmd/app.go

```diff
@@ -64,13 +52,6 @@ func (app *App) PrepareConfig() error {
 
 	app.Config.Services.Clean()
 
-	if app.Config.Settings.SkipShuffle {
-		log.Infof("Skip shuffeling service order.")
-	} else {
-		log.Infof("Shuffling service list")
-		app.Config.Services.Shuffle()
-	}
```

Comment:
or in other words: are we sure that the order of deployments doesn't matter for things to get into a working condition?